render/egl.c

```diff
@@ -101,6 +102,28 @@ static bool check_egl_ext(const char *egl_exts, const char *ext) {
 	return false;
 }
 
+static void print_dmabuf_formats(struct wlr_egl *egl)
+{
```

Comment:
Style nit: these two lines should be on the same one (I understand it's difficult to switch between wlroots' style and weston's :P ).